FirebaseSessions/Tests/TestApp/UITestsiOS/UITestsiOS.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+final class UITestsiOS: XCTestCase {
+  override func setUpWithError() throws {
```

Comment:
Both the `setup` and `tearDown` methods can be cleaned up as we are not using them.